transmart-notifications/grails-app/conf/plugin.yml

```diff
@@ -2,10 +2,16 @@
 # max number of query sets returned in a subscription email
 org.transmart.notifications.maxNumberOfSets: 20
 
+# daily cron job trigger time in format: hh-mm
+# hh: Hour, range: 0-23;
+# mm: Minute, range: 0-59;
+org.transmart.notifications.dailyJobTriggerTime: 0-0
+
 # Quartz plugin configuration - job for query subscription
 # To disable the email sending job, change this to false
 quartz:
-    jobEnabled: false
+    autoStartup: true
+    jobEnabled: true
```

Comment:
Why isn't this configuration option named something like `org.transmartproject.notifications.enabled = true`?

Reply:
ah good one, I forgot that `jobEnabled` is not a quartz plugin config option. Currently it's not even used, I will change it.